test/nodes/test_audio.py

```diff
@@ -45,6 +45,7 @@ def whisper_helper():
     return WhisperHelper("openai/whisper-medium")
 
 
+@pytest.mark.integration
```

Comment:
I am assuming this will eventually be moved into e2e, and this is a step towards the same.